django_spanner/__init__.py

```diff
@@ -34,22 +33,24 @@
 if django.VERSION[:2] == (3, 2):
     USING_DJANGO_3 = True
 
-if USING_DJANGO_3:
-    from django.db.models.fields import (
-        SmallAutoField,
-        BigAutoField,
-    )
-    from django.db.models import JSONField
+USING_DJANGO_4 = False
+if django.VERSION[:2] == (4, 2):
```

Comment:
as commented in this draft:
https://github.com/googleapis/python-spanner-django/pull/860#discussion_r1531969853

Currently Django 5 is released, since you are already working on it, can this field be override with a system environment, and make it something like:
`if django.VERSION[:2] == (4, 2) or os.env.get('USING_DJANGO_4', `false`) == 'true'`

Reply:
Sorry we cant do that as we are not sure how will Django 5 behave with this code. 
Please feel free to create a fork and make changes

Reply:
Can I second @jaz-la's request? If we can appropriately set expectations with the env car, what's the harm? I am currently using a fork that is hard to maintain with Django 5, and this PR includes everything that I have needed for Django 5 so far. Please? 

Thanks again for all of your work.